src/app/shared/services/message.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+// import { catchError, map, tap } from 'rxjs/operators';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+
+
+export class MessageService {
+  private alert = new BehaviorSubject<object>({});
```

Comment:
the same - why you don`t use directly alert, but create a new variable currentMassage?

Reply:
fixed